src/elements/content-sidebar/__tests__/ActivitySidebar.test.js

```diff
@@ -356,21 +356,29 @@ describe('elements/content-sidebar/ActivitySidebar', () => {
         });
 
         test.each`
-            annotationsEnabled | appActivityEnabled | expectedAnnotations | expectedAppActivity
-            ${false}           | ${false}           | ${false}            | ${false}
-            ${false}           | ${true}            | ${false}            | ${true}
-            ${true}            | ${false}           | ${true}             | ${false}
-            ${true}            | ${true}            | ${true}             | ${true}
+            annotationsEnabled | appActivityEnabled | versionsEnabled | expectedAnnotations | expectedAppActivity | expectedVersions
+            ${false}           | ${false}           | ${false}        | ${false}            | ${false}            | ${false}
+            ${false}           | ${true}            | ${false}        | ${false}            | ${true}             | ${false}
+            ${true}            | ${false}           | ${true}         | ${true}             | ${false}            | ${true}
+            ${true}            | ${true}            | ${true}         | ${true}             | ${true}             | ${true}
```

Comment:
There are some more cases if we want to be exhaustive:
```
            annotationsEnabled | appActivityEnabled | versionsEnabled
            ${false}           | ${false}           | ${true}
            ${false}           | ${true}            | ${true}
            ${true}            | ${false}           | ${false}
            ${true}            | ${true}            | ${false}
```

Reply:
Technically those features do not depend on each other, but I've added missing combinations.